bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/NAThing.java

```diff
@@ -52,7 +54,14 @@ public ModuleType getType() {
     public boolean isReachable() {
         // This is not implemented on all devices/modules, so if absent we consider it is reachable
         Boolean localReachable = this.reachable;
-        return localReachable != null ? localReachable : true;
+        boolean result = localReachable != null ? localReachable : true;
+        // and we double check by comparing data freshness
+        ZonedDateTime localLastSeen = lastSeen;
+        if (result && localLastSeen != null) {
+            result = Duration.between(localLastSeen, ZonedDateTime.now().withZoneSameInstant(localLastSeen.getZone()))
+                    .getSeconds() < UNREACHABLE_DELAY_S;
+        }
```

Comment:
Could this be simplified by using `Instant` since logic doesn't seem to care about time-zone?

Reply:
Done